tests/helpers.py

```
@@ -44,57 +43,36 @@ def make_date_dt(date, tz):
+def make_client():
+    """Return a fresh ACWDClient instance."""
+    return ACWDClient("user@example.com", "secret")
 
 
-def load_stats_module():
-    """Load the statistics module without triggering __init__.py.
+def make_logged_in_client(meter_cached=True):
+    """Return a logged-in ACWDClient with optional pre-cached meter."""
+    client = make_client()
+    client.logged_in = True
+    client.csrf_token = "tok123"
```

Ruff (0.15.7)

[error] Possible hardcoded password assigned to: "csrf_token"

(S105)